src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./Breadcrumb.module.css"
+
+interface LinkProps {
+  href: string
+  isCurrent: boolean
+  label: string
+}
+
+interface Props {
+  links: LinkProps[]
+}
+
+export function BreadcrumbItem({ links }: Props) {
```

Comment:
@belv2c Looking good overall! 

Maybe I'm getting this wrong but the story for this component suggests that the ideal utilization is to have:
```
1 Breadcrumb -> 1 BreadcrumbItem -> Many links
```

I would think that a breadcrumb would have a collection of links (or items), somewhat similar to how `Tabs` have a collection of `Tab` components.

Is there a case for having a `BreadcrumbItem` take a collection of `link`s? For example, would it make sense to have several `BreadcrumbItem`s and each one of them have their own list of `links`? 🤔 

I'm thinking the straightforward path is to have the `BreadcrumbItem` _be_ one link, so then you can enforce that a `Breadcrumb` _should_ only have `BreadcrumbItem`s as children. (as opposed to any `ReactNode`)

Sorry if this was very confusing, I can try to explain my point better

Reply:
I agree with what @luiz-chagas is pointing out here. I wouldn't expect an "item" to produce a "list"—I expect a "list" to produce "items". If that makes sense... 😆

Reply:
@luiz-chagas @roblafeve This makes sense - I was struggling with some naming variations `links` `link` `item` `items` 😂. I've set aside some time tomorrow to resolve this PR feedback!